docs/_providers/akamaiedgedns.md

```diff
@@ -0,0 +1,68 @@
+---
+name: AkamaiEdgeDns
+title: Akamai Edge DNS Provider
+layout: default
+jsId: AKAMAIEDGEDNS
+---
+# Akamai Edge DNS Provider
```

Comment:
Please leave a blank line before and after any `#` and before and after any code fence (three ` in a row)

Reply:
Done